pootle/apps/pootle_app/management/commands/refresh_scores.py

```diff
@@ -28,10 +29,46 @@ def add_arguments(self, parser):
             default=False,
             help='Reset all scores to zero',
         )
+        parser.add_argument(
+            '--user',
+            action='append',
+            dest='users',
+            help='User to refresh',
+        )
 
     def handle(self, **options):
+        start = datetime.datetime.now()
+        self.stdout.write('Start running of refresh_scores command...')
+
+        User = get_user_model()
+        users = User.objects.all()
+        if options['users']:
+            users = users.filter(username__in=options['users'])
 
         if options['reset']:
-            User = get_user_model()
-            User.objects.all().update(score=0)
-            ScoreLog.objects.all().delete()
+            users.update(score=0)
+            scorelogs = ScoreLog.objects.all()
+            if options['users']:
+                scorelogs = scorelogs.filter(user__in=users)
+            scorelogs.delete()
+            self.stdout.write('All user scores were reset to 0.')
```

Comment:
I don't think we need to list the users since those we specified. I would use:

```python
            if options['users']:
                self.stdout.write('Specified users scores were reset to 0.')
            else:
                self.stdout.write('All users scores were reset to 0.')
```